cppcache/src/TcrMessage.cpp

```diff
@@ -388,17 +707,6 @@ inline void TcrMessage::readKeyPart(DataInput& input) {
   }
 }
 
-inline void TcrMessage::writeInt(uint8_t* buffer, uint16_t value) {
```

Comment:
Now the compiler can’t inline these frequently executed methods.

Reply:
Yes it can. An optimizing compiler is free to inline this function given the entire definition is visible and constrained to this source file.

Reply:
Ah, didn't see that it was yanked from private.